### PR TITLE
Actuator for ingest and MIS services

### PR DESCRIPTION
**Pull request by ahoffer**
Adds in dependencies and configurations for Spring Boot Actuator. 
Configured to expose all Actuator endpoints.

To test (using *local deployment*), enter an actuator endpoint into a browser. E.g.

`http://127.0.0.1:9040/actuator/httptrace`
`http://127.0.0.1:9041/actuator`

### CODE REVIEW

**Comment by ahoffer** [comment]
build now

File: buildSrc/src/main/groovy/Dependencies.groovy [hunk 1]
```diff
@@ -16,6 +16,8 @@ class Dependencies {
   static def springBootStarterWeb = { it=Versions.springBoot -> "org.springframework.boot:spring-boot-starter-web:${it}" }
   static def springDataSolr = { it=Versions.springData -> "org.springframework.data:spring-data-solr:${it}" }
   static def transformAPI = { it=Versions.transformAPI -> "com.connexta.transformation:transformation-api-rest-spring-stubs:${it}" }
+  static def springActuator = { it=Versions.springBoot -> "org.springframework.boot:spring-boot-starter-actuator:${it}" }
```
Comment (jhunzik) [inline buildSrc/src/main/groovy/Dependencies.groovy]: ❓ I wonder if we need to lock this endpoint now at this point.

File: multi-int-store/src/main/resources/application.yml [hunk 2]
```diff
@@ -8,7 +8,6 @@ management:
     web:
       exposure:
         include: "*"
```
Comment (jhunzik) [inline multi-int-store/src/main/resources/application.yml]: I wonder if we should include all the endpoints or maybe just httptrace, health and loggers.

File: buildSrc/src/main/groovy/Dependencies.groovy (hunk 1 above)
Comment (ahoffer) [reply buildSrc/src/main/groovy/Dependencies.groovy]: Eventually, yes. It is not something you want exposed in a production system. We can do that Until we integrate with the security services.

File: multi-int-store/src/main/resources/application.yml (hunk 2 above)
Comment (ahoffer) [reply multi-int-store/src/main/resources/application.yml]: I don't know which endpoints will prove useful yet. I would like to leave them open and play around with them a bit.

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Pull request merged**